trunk/DefaultCombat/Routines/Advanced/Guardian/Defense.cs

```diff
@@ -1,9 +1,14 @@
 ﻿// Copyright (C) 2011-2017 Bossland GmbH
 // See the file LICENSE for the source code's detailed license
 
+using System.Windows.Input;
 using Buddy.BehaviorTree;
+using Buddy.CommonBot;
+using Buddy.Swtor;
+using Buddy.Swtor.Objects;
 using DefaultCombat.Core;
 using DefaultCombat.Helpers;
+using Targeting = DefaultCombat.Core.Targeting;

```

Comment:
What is this conflicting with? Afaik there aren't multiple `Targeting` types, is it clashing between routine and bot core?